extensions/amp-carousel/0.1/test/validator-amp-carousel.out

```diff
@@ -2,3 +2,4 @@ FAIL
 amp-carousel/0.1/test/validator-amp-carousel.html:31:2 The attribute 'delay' in tag 'amp-carousel' is set to the invalid value ''. (see https://www.ampproject.org/docs/reference/components/amp-carousel) [AMP_TAG_PROBLEM]
 amp-carousel/0.1/test/validator-amp-carousel.html:34:2 The tag 'amp-carousel' requires including the 'amp-carousel' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-carousel) [AMP_TAG_PROBLEM]
 amp-carousel/0.1/test/validator-amp-carousel.html:37:2 The tag 'amp-carousel' requires including the 'amp-carousel' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-carousel) [AMP_TAG_PROBLEM]
+amp-carousel/0.1/test/validator-amp-carousel.html:40:2 The tag 'amp-carousel' requires including the 'amp-carousel' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-carousel) [AMP_TAG_PROBLEM]
```

Comment:
Same here for `amp-carousel`
`<script async custom-element="amp-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.1.js"></script>`